lib/src/components/NxSearchDropdown/NxSearchDropdown.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
```

Comment:
Just wondering if we should be using `@use` instead of `@import` since we moved to dart-sass?